scalameter-core/src/main/scala/org/scalameter/execution/invocation/InvocationCountMatcher.scala

```diff
@@ -79,14 +80,18 @@ object InvocationCountMatcher {
       def matches(methodName: String, methodDescriptor: String): Boolean = regex.matcher(methodName).matches()
     }
 
-    /** Matches method with a [[java.lang.reflect.Method]].
+    /**  Matches method with a [[java.lang.reflect.Method]].
```

Comment:
Please update this comment in case you no longer wish to use `Method` as ctor param.